### PR TITLE
Make sure verifyCapabilities build steps are tied to producing a build item

### PR DESCRIPTION
**Pull request by aloubyansky**
W/o this a build step might not be executed and actually lead to an application build error.

### CODE REVIEW

**Comment by aloubyansky** [comment]
@radtriste this is a critical one, pretty much a blocker. Could you guys have a look into this?

**Comment by danielezonca** [comment]
ok to test

File: quarkus/extensions/kogito-quarkus-extension-common/kogito-quarkus-common-deployment/src/main/java/org/kie/kogito/quarkus/common/deployment/KogitoAssetsProcessor.java [hunk 1]
```diff
@@ -115,6 +117,7 @@ public SyntheticBeanBuildItem runtimeConfigBuildStep(KogitoRecorder recorder) {
     }
 
     @BuildStep
+    @Produce(EmptyBuildItem.class)
```
Comment (danielezonca) [inline quarkus/extensions/kogito-quarkus-extension-common/kogito-quarkus-common-deployment/src/main/java/org/kie/kogito/quarkus/common/deployment/KogitoAssetsProcessor.java:120]: Why this change? This step produce a `KogitoGeneratedSourcesBuildItem` so it is not empty

Comment (aloubyansky) [reply quarkus/extensions/kogito-quarkus-extension-common/kogito-quarkus-common-deployment/src/main/java/org/kie/kogito/quarkus/common/deployment/KogitoAssetsProcessor.java:120]: Right, I'll remove this one. Thanks!

**Comment by kie-ci3** [comment]
**(tests) - kogito-examples job** `#684` was: **FAILURE**
Possible explanation: Pipeline failure or project build failure

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-runtimes.tests.downstream.kogito-examples/684/display/redirect
See console log:
<details>
<summary><b>Console Logs</b></summary>
Checking out Revision 34c2cbeaf58086cb3d2eed257b72b22b17ade321 (main)<br/> > git config core.sparsecheckout # timeout=10<br/> > git checkout -f 34c2cbeaf58086cb3d2eed257b72b22b17ade321 # timeout=10<br/>Commit message: "MavenCommand: Use Maven Wrapper (#211)"<br/> > git rev-list --no-walk 34c2cbeaf58086cb3d2eed257b72b22b17ade321 # timeout=10<br/>[Pipeline] Start of Pipeline<br/>[Pipeline] node<br/>Running on rhos-01-kie-psi-rhel7-xlarge-549 in /home/jenkins/workspace/KIE/kogito/main/pullrequest/kogito-runtimes.tests.downstream.kogito-examples<br/>[Pipeline] {<br/>[Pipeline] stage<br/>[Pipeline] { (Declarative: Checkout SCM)<br/>[Pipeline] checkout<br/>The recommended git tool is: NONE<br/>using credential kie-ci<br/>Cloning the remote Git repository<br/>Avoid second fetch<br/>Checking out Revision 49ff30e440728e9764001756b15e778d6c4c4b17 (origin/main)<br/>Commit message: "Add release branch 1.26.x after cut-off (#581)"<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] withEnv<br/>[Pipeline] {<br/>[Pipeline] withEnv<br/>[Pipeline] {<br/>[Pipeline] timestamps<br/>[Pipeline] {<br/>[Pipeline] timeout<br/>[2022-08-11T09:10:25.323Z] Timeout set to expire in 3 hr 0 min<br/>[Pipeline] {<br/>[Pipeline] stage<br/>[Pipeline] { (Declarative: Tool Install)<br/>[Pipeline] tool<br/>[2022-08-11T09:10:25.435Z] Installing NodeJS from /var/lib/jenkins/caches/nodejs/LINUX/amd64/16.2.0.tar.gz to /home/jenkins/tools/jenkins.plugins.nodejs.tools.NodeJSInstallation/nodejs-16.2.0 on rhos-01-kie-psi-rhel7-xlarge-549<br/>Cloning repository https://github.com/kiegroup/kogito-pipelines.git<br/> > git init /home/jenkins/workspace/KIE/kogito/main/pullrequest/kogito-runtimes.tests.downstream.kogito-examples # timeout=10<br/>Fetching upstream changes from https://github.com/kiegroup/kogito-pipelines.git<br/> > git --version # timeout=10<br/> > git --version # 'git version 2.37.1'<br/>using GIT_ASKPASS to set credentials PSSWD: kie-ci service account with access to jboss-integration private github space<br/> > git fetch --tags --force --progress -- https://github.com/kiegroup/kogito-pipelines.git +refs/heads/*:refs/remotes/origin/* # timeout=10<br/> > git config remote.origin.url https://github.com/kiegroup/kogito-pipelines.git # timeout=10<br/> > git config --add remote.origin.fetch +refs/heads/*:refs/remotes/origin/* # timeout=10<br/> > git rev-parse origin/main^{commit} # timeout=10<br/> > git config core.sparsecheckout # timeout=10<br/> > git checkout -f 49ff30e440728e9764001756b15e778d6c4c4b17 # timeout=10<br/>[Pipeline] envVarsForTool<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] withEnv<br/>[Pipeline] {<br/>[Pipeline] stage<br/>[Pipeline] { (Initialize)<br/>[Pipeline] tool<br/>[Pipeline] envVarsForTool<br/>[Pipeline] withEnv<br/>[Pipeline] {<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>[2022-08-11T09:10:28.118Z] + printenv<br/>[Pipeline] archiveArtifacts<br/>[2022-08-11T09:10:28.142Z] Archiving artifacts<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] }<br/>[Pipeline] // withEnv<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (check space before build)<br/>[Pipeline] tool<br/>[Pipeline] envVarsForTool<br/>[Pipeline] withEnv<br/>[Pipeline] {<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] dir<br/>[2022-08-11T09:10:28.612Z] Running in /home/jenkins/workspace/KIE/kogito/main/pullrequest/kogito-runtimes.tests.downstream.kogito-examples<br/>[Pipeline] {<br/>[Pipeline] echo<br/>[2022-08-11T09:10:28.642Z] [INFO] space left on the machine<br/>[Pipeline] sh<br/>[2022-08-11T09:10:28.938Z] + df -h<br/>[2022-08-11T09:10:28.938Z] Filesystem      Size  Used Avail Use% Mounted on<br/>[2022-08-11T09:10:28.938Z] devtmpfs        7.8G     0  7.8G   0% /dev<br/>[2022-08-11T09:10:28.938Z] tmpfs           7.8G     0  7.8G   0% /dev/shm<br/>[2022-08-11T09:10:28.938Z] tmpfs           7.8G   17M  7.8G   1% /run<br/>[2022-08-11T09:10:28.938Z] tmpfs           7.8G     0  7.8G   0% /sys/fs/cgroup<br/>[2022-08-11T09:10:28.938Z] /dev/vda1        80G   19G   62G  23% /<br/>[2022-08-11T09:10:28.938Z] tmpfs           1.6G     0  1.6G   0% /run/user/1000<br/>[Pipeline] echo<br/>[2022-08-11T09:10:28.957Z] [INFO] space of /home/jenkins<br/>[Pipeline] sh<br/>[2022-08-11T09:10:29.240Z] + du -h -d1 /home/jenkins<br/>[2022-08-11T09:10:29.240Z] 4.0K	/home/jenkins/.ssh<br/>[2022-08-11T09:10:29.240Z] 0	/home/jenkins/go<br/>[2022-08-11T09:10:29.240Z] 8.0K	/home/jenkins/.vnc<br/>[2022-08-11T09:10:29.240Z] 8.0K	/home/jenkins/.m2<br/>[2022-08-11T09:10:29.240Z] 2.1G	/home/jenkins/git-repos<br/>[2022-08-11T09:10:29.240Z] 8.0K	/home/jenkins/.config<br/>[2022-08-11T09:10:29.240Z] 4.0K	/home/jenkins/.cekit<br/>[2022-08-11T09:10:29.496Z] 82M	/home/jenkins/virtenvs<br/>[2022-08-11T09:10:29.496Z] 31M	/home/jenkins/remoting<br/>[2022-08-11T09:10:29.496Z] 0	/home/jenkins/.cache<br/>[2022-08-11T09:10:29.496Z] 2.7M	/home/jenkins/workspace<br/>[2022-08-11T09:10:29.496Z] 0	/home/jenkins/.pki<br/>[2022-08-11T09:10:29.496Z] 0	/home/jenkins/caches<br/>[2022-08-11T09:10:29.496Z] 8.0K	/home/jenkins/.java<br/>[2022-08-11T09:10:29.496Z] 100M	/home/jenkins/tools<br/>[2022-08-11T09:10:29.496Z] 2.3G	/home/jenkins<br/>[Pipeline] echo<br/>[2022-08-11T09:10:29.515Z] [INFO] space of workspace<br/>[Pipeline] sh<br/>[2022-08-11T09:10:29.810Z] + du -h -d3 /home/jenkins/workspace<br/>[2022-08-11T09:10:29.810Z] 0	/home/jenkins/workspace/KIE/optaplanner/main<br/>[2022-08-11T09:10:29.810Z] 0	/home/jenkins/workspace/KIE/optaplanner<br/>[2022-08-11T09:10:29.810Z] 2.7M	/home/jenkins/workspace/KIE/kogito/main<br/>[2022-08-11T09:10:29.810Z] 2.7M	/home/jenkins/workspace/KIE/kogito<br/>[2022-08-11T09:10:29.810Z] 2.7M	/home/jenkins/workspace/KIE<br/>[2022-08-11T09:10:29.810Z] 2.7M	/home/jenkins/workspace<br/>[Pipeline] }<br/>[Pipeline] // dir<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] }<br/>[Pipeline] // withEnv<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Install build-chain tool)<br/>[Pipeline] tool<br/>[Pipeline] envVarsForTool<br/>[Pipeline] withEnv<br/>[Pipeline] {<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] echo<br/>[2022-08-11T09:10:30.140Z] [INFO] Getting build-chain version from composite action file<br/>[Pipeline] readFile<br/>[Pipeline] echo<br/>[2022-08-11T09:10:30.231Z] [INFO] build-chain version recovered '2.3.34'<br/>[Pipeline] sh<br/>[2022-08-11T09:10:30.515Z] + npm install -g @kie/build-chain-action@2.3.34 -registry=https://repository.engineering.redhat.com/nexus/repository/registry.npmjs.org/<br/>[2022-08-11T09:10:31.442Z] npm WARN ERESOLVE overriding peer dependency<br/>[2022-08-11T09:10:31.442Z] npm WARN Found: @octokit/core@2.5.4<br/>[2022-08-11T09:10:31.442Z] npm WARN node_modules/@kie/build-chain-action/node_modules/@octokit/core<br/>[2022-08-11T09:10:31.442Z] npm WARN   @octokit/core@"^2.4.3" from @octokit/rest@17.11.2<br/>[2022-08-11T09:10:31.442Z] npm WARN   node_modules/@kie/build-chain-action/node_modules/@octokit/rest<br/>[2022-08-11T09:10:31.442Z] npm WARN     @octokit/rest@"^17.6.0" from @kie/build-chain-action@2.3.34<br/>[2022-08-11T09:10:31.442Z] npm WARN     node_modules/@kie/build-chain-action<br/>[2022-08-11T09:10:31.442Z] npm WARN     1 more (@kie/build-chain-configuration-reader)<br/>[2022-08-11T09:10:31.442Z] npm WARN   1 more (@octokit/plugin-paginate-rest)<br/>[2022-08-11T09:10:31.442Z] npm WARN <br/>[2022-08-11T09:10:31.442Z] npm WARN Could not resolve dependency:<br/>[2022-08-11T09:10:31.442Z] npm WARN peer @octokit/core@">=3" from @octokit/plugin-request-log@1.0.4<br/>[2022-08-11T09:10:31.442Z] npm WARN node_modules/@kie/build-chain-action/node_modules/@octokit/plugin-request-log<br/>[2022-08-11T09:10:31.442Z] npm WARN   @octokit/plugin-request-log@"^1.0.0" from @octokit/rest@17.11.2<br/>[2022-08-11T09:10:31.442Z] npm WARN   node_modules/@kie/build-chain-action/node_modules/@octokit/rest<br/>[2022-08-11T09:10:34.706Z] <br/>[2022-08-11T09:10:34.706Z] added 111 packages, and audited 112 packages in 4s<br/>[2022-08-11T09:10:34.706Z] <br/>[2022-08-11T09:10:34.706Z] 5 packages are looking for funding<br/>[2022-08-11T09:10:34.706Z]   run `npm fund` for details<br/>[2022-08-11T09:10:34.706Z] <br/>[2022-08-11T09:10:34.706Z] found 0 vulnerabilities<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] }<br/>[Pipeline] // withEnv<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Build projects)<br/>[Pipeline] tool<br/>[Pipeline] envVarsForTool<br/>[Pipeline] tool<br/>[Pipeline] envVarsForTool<br/>[Pipeline] tool<br/>[Pipeline] envVarsForTool<br/>[Pipeline] withEnv<br/>[Pipeline] {<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] echo<br/>[2022-08-11T09:10:35.168Z] BUILD_MVN_OPTS_CURRENT = <br/>[Pipeline] configFileProvider<br/>[2022-08-11T09:10:35.203Z] provisioning config files...<br/>[2022-08-11T09:10:35.214Z] copy managed file [kogito_pr_settings] to file:/home/jenkins/workspace/KIE/kogito/main/pullrequest/kogito-runtimes.tests.downstream.kogito-examples@tmp/config7951703569256696033tmp<br/>[Pipeline] {<br/>[Pipeline] withCredentials<br/>[2022-08-11T09:10:35.255Z] Masking supported pattern matches of $GITHUB_TOKEN<br/>[Pipeline] {<br/>[Pipeline] sh<br/>[2022-08-11T09:10:35.289Z] Warning: A secret was passed to "sh" using Groovy String interpolation, which is insecure.<br/>[2022-08-11T09:10:35.290Z] 		 Affected argument(s) used the following variable(s): [GITHUB_TOKEN]<br/>[2022-08-11T09:10:35.290Z] 		 See https://jenkins.io/redirect/groovy-string-interpolation for details.<br/>[2022-08-11T09:10:35.563Z] + build-chain-action -token=**** -df=https://raw.githubusercontent.com/kiegroup/kogito-pipelines/main/.ci/pull-request-config.yaml -folder=bc build pr -url=https://github.com/kiegroup/kogito-runtimes/pull/2388 -sp=kiegroup/kogito-examples --skipParallelCheckout -cct '(^mvn .*)||$1 -s /home/jenkins/workspace/KIE/kogito/main/pullrequest/kogito-runtimes.tests.downstream.kogito-examples@tmp/config7951703569256696033tmp -Dmaven.wagon.http.ssl.insecure=true -Dmaven.test.failure.ignore=true'<br/>[2022-08-11T09:10:36.123Z] [INFO]  Executing pull request flow for https://github.com/kiegroup/kogito-runtimes/pull/2388 in bc<br/>[2022-08-11T09:10:36.124Z] (node:26053) Warning: Accessing non-existent property 'ClientError' of module exports inside circular dependency<br/>[2022-08-11T09:10:36.124Z] (Use `node --trace-warnings ...` to show where the warning was created)<br/>[2022-08-11T09:10:36.124Z] [INFO]  https://raw.githubusercontent.com/kiegroup/kogito-pipelines/main/.ci/pull-request-config.yaml exists, using it<br/>[2022-08-11T09:10:36.400Z] ::group::[PRE] Executing pre section for https://raw.githubusercontent.com/kiegroup/kogito-pipelines/main/.ci/pull-request-config.yaml<br/>[2022-08-11T09:10:36.400Z] [INFO]  Execute command 'export BUILD_MVN_OPTS=" -nsu -ntp -fae -e -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false dependency:tree -Dmaven.wagon.httpconnectionManager.ttlSeconds=120 -Dmaven.wagon.http.retryHandler.count=3"' in dir '/home/jenkins/workspace/KIE/kogito/main/pullrequest/kogito-runtimes.tests.downstream.kogito-examples'<br/>[2022-08-11T09:10:36.400Z] [INFO]  Treating export command since it's not possible to run it from the runner itself<br/>[2022-08-11T09:10:36.400Z] [INFO]  The variable BUILD_MVN_OPTS has been stored with ' -nsu -ntp -fae -e -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false dependency:tree -Dmaven.wagon.httpconnectionManager.ttlSeconds=120 -Dmaven.wagon.http.retryHandler.count=3' value<br/>[2022-08-11T09:10:36.400Z] ::notice title= [PRE] [Execution OK]::export BUILD_MVN_OPTS=" -nsu -ntp -fae -e -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false dependency:tree -Dmaven.wagon.httpconnectionManager.ttlSeconds=120 -Dmaven.wagon.http.retryHandler.count=3"<br/>[2022-08-11T09:10:36.400Z] [INFO]  Execute command 'echo "BUILD_MVN_OPTS= -nsu -ntp -fae -e -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false dependency:tree -Dmaven.wagon.httpconnectionManager.ttlSeconds=120 -Dmaven.wagon.http.retryHandler.count=3"' in dir '/home/jenkins/workspace/KIE/kogito/main/pullrequest/kogito-runtimes.tests.downstream.kogito-examples'<br/>[2022-08-11T09:10:36.400Z] [command]/usr/bin/echo BUILD_MVN_OPTS= -nsu -ntp -fae -e -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false dependency:tree -Dmaven.wagon.httpconnectionManager.ttlSeconds=120 -Dmaven.wagon.http.retryHandler.count=3<br/>[2022-08-11T09:10:36.400Z] BUILD_MVN_OPTS= -nsu -ntp -fae -e -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false dependency:tree -Dmaven.wagon.httpconnectionManager.ttlSeconds=120 -Dmaven.wagon.http.retryHandler.count=3<br/>[2022-08-11T09:10:36.400Z] ::notice title= [PRE] [Execution OK]::echo "BUILD_MVN_OPTS= -nsu -ntp -fae -e -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false dependency:tree -Dmaven.wagon.httpconnectionManager.ttlSeconds=120 -Dmaven.wagon.http.retryHandler.count=3"<br/>[2022-08-11T09:10:36.400Z] [INFO]  Execute command 'export BUILD_MVN_OPTS_CURRENT=""' in dir '/home/jenkins/workspace/KIE/kogito/main/pullrequest/kogito-runtimes.tests.downstream.kogito-examples'<br/>[2022-08-11T09:10:36.400Z] [INFO]  Treating export command since it's not possible to run it from the runner itself<br/>[2022-08-11T09:10:36.400Z] [INFO]  The variable BUILD_MVN_OPTS_CURRENT has been stored with '' value<br/>[2022-08-11T09:10:36.400Z] ::notice title= [PRE] [Execution OK]::export BUILD_MVN_OPTS_CURRENT=""<br/>[2022-08-11T09:10:36.400Z] [INFO]  Execute command 'echo "BUILD_MVN_OPTS_CURRENT="' in dir '/home/jenkins/workspace/KIE/kogito/main/pullrequest/kogito-runtimes.tests.downstream.kogito-examples'<br/>[2022-08-11T09:10:36.400Z] [command]/usr/bin/echo BUILD_MVN_OPTS_CURRENT=<br/>[2022-08-11T09:10:36.400Z] BUILD_MVN_OPTS_CURRENT=<br/>[2022-08-11T09:10:36.400Z] ::notice title= [PRE] [Execution OK]::echo "BUILD_MVN_OPTS_CURRENT="<br/>[2022-08-11T09:10:36.400Z] ::endgroup::<br/>[2022-08-11T09:10:36.400Z] ::group::[Pull Request Flow] Execution Plan...<br/>[2022-08-11T09:10:36.400Z] [INFO]  ----------------------------------------------<br/>[2022-08-11T09:10:36.400Z] [INFO]  [4] projects will be executed<br/>[2022-08-11T09:10:36.400Z] <br/>[2022-08-11T09:10:36.400Z] [INFO]  [kiegroup/drools]<br/>[2022-08-11T09:10:36.400Z] [INFO]  Level Type: [upstream].<br/>[2022-08-11T09:10:36.400Z] [INFO]  mvn clean install -Dquickly ${{ env.BUILD_MVN_OPTS }} ${{ env.BUILD_MVN_OPTS_UPSTREAM }} ${{ env.DROOLS_BUILD_MVN_OPTS_UPSTREAM }}<br/>[2022-08-11T09:10:36.400Z] [INFO]  [ [32m'docker system prune -f'[39m ]<br/>[2022-08-11T09:10:36.400Z] <br/>[2022-08-11T09:10:36.400Z] [INFO]  [kiegroup/kogito-runtimes]<br/>[2022-08-11T09:10:36.400Z] [INFO]  Level Type: [upstream].<br/>[2022-08-11T09:10:36.400Z] [INFO]  mvn clean install -Dquickly ${{ env.BUILD_MVN_OPTS }} ${{ env.BUILD_MVN_OPTS_UPSTREAM }} ${{ env.KOGITO_RUNTIMES_BUILD_MVN_OPTS_UPSTREAM }}<br/>[2022-08-11T09:10:36.400Z] [INFO]  [ [32m'docker system prune -f'[39m ]<br/>[2022-08-11T09:10:36.400Z] <br/>[2022-08-11T09:10:36.400Z] [INFO]  [kiegroup/kogito-apps]<br/>[2022-08-11T09:10:36.400Z] [INFO]  Level Type: [upstream].<br/>[2022-08-11T09:10:36.400Z] [INFO]  mvn clean install -Dquickly ${{ env.BUILD_MVN_OPTS }} ${{ env.BUILD_MVN_OPTS_UPSTREAM }} ${{ env.KOGITO_APPS_BUILD_MVN_OPTS_UPSTREAM }}<br/>[2022-08-11T09:10:36.400Z] [INFO]  [ [32m'docker system prune -f'[39m ]<br/>[2022-08-11T09:10:36.400Z] <br/>[2022-08-11T09:10:36.400Z] [INFO]  [kiegroup/kogito-examples]<br/>[2022-08-11T09:10:36.401Z] [INFO]  Level Type: [current].<br/>[2022-08-11T09:10:36.401Z] [INFO]  [<br/>[2022-08-11T09:10:36.401Z]   [32m'mvn -f kogito-examples/pom.xml clean install -Dvalidate-formatting ${{ env.BUILD_MVN_OPTS }} ${{ env.BUILD_MVN_OPTS_CURRENT }} ${{ env.KOGITO_EXAMPLES_BUILD_MVN_OPTS }}'[39m,<br/>[2022-08-11T09:10:36.401Z]   [32m'bash -c "if [ \'${{ env.DISABLE_PERSISTENCE }}\' != \'true\' ]; then mvn -f kogito-examples-persistence/pom.xml clean install -Ppersistence ${{ env.BUILD_MVN_OPTS }} ${{ env.BUILD_MVN_OPTS_CURRENT }} ${{ env.KOGITO_EXAMPLES_BUILD_MVN_OPTS }}; fi"'[39m,<br/>[2022-08-11T09:10:36.401Z]   [32m'bash -c "if [ \'${{ env.DISABLE_EVENTS }}\' != \'true\' ]; then mvn -f kogito-examples-events/pom.xml clean install -Pevents ${{ env.BUILD_MVN_OPTS }} ${{ env.BUILD_MVN_OPTS_CURRENT }} ${{ env.KOGITO_EXAMPLES_BUILD_MVN_OPTS }}; fi"'[39m<br/>[2022-08-11T09:10:36.401Z] ]<br/>[2022-08-11T09:10:36.401Z] [INFO]  [ [32m'docker system prune -f'[39m ]<br/>[2022-08-11T09:10:36.401Z] <br/>[2022-08-11T09:10:36.401Z] [INFO]  ----------------------------------------------<br/>[2022-08-11T09:10:36.401Z] ::endgroup::<br/>[2022-08-11T09:10:36.401Z] ::group::[Pull Request Flow] Checking out kiegroup/kogito-runtimes and its dependencies<br/>[2022-08-11T09:10:36.401Z] [INFO]  Tree for project kiegroup/kogito-examples. Dependencies: <br/>[2022-08-11T09:10:36.401Z] kiegroup/drools,<br/>[2022-08-11T09:10:36.401Z] kiegroup/kogito-runtimes,<br/>[2022-08-11T09:10:36.401Z] kiegroup/kogito-apps,<br/>[2022-08-11T09:10:36.401Z] kiegroup/kogito-examples<br/>[2022-08-11T09:10:36.401Z] [INFO]  [kiegroup/drools] Checking out project<br/>[2022-08-11T09:10:36.657Z] [WARN]  aloubyansky/drools not found. Trying to get it by forked project list.<br/>[2022-08-11T09:11:32.840Z] [ERROR]  You have exceeded a secondary rate limit. Please wait a few minutes before you try again.<br/>[Pipeline] }<br/>[Pipeline] // withCredentials<br/>[Pipeline] }<br/>[2022-08-11T09:11:32.907Z] Deleting 1 temporary files<br/>[Pipeline] // configFileProvider<br/>[Pipeline] }<br/>[Pipeline] // script<br/>Post stage<br/>[Pipeline] junit<br/>[2022-08-11T09:11:33.018Z] Recording test results<br/>[2022-08-11T09:11:33.056Z] No test report files were found. Configuration error?<br/>Error when executing always post condition:<br/>Also:   hudson.remoting.Channel$CallSiteStackTrace: Remote call to rhos-01-kie-psi-rhel7-xlarge-549<br/>		at hudson.remoting.Channel.attachCallSiteStackTrace(Channel.java:1797)<br/>		at hudson.remoting.UserRequest$ExceptionResponse.retrieve(UserRequest.java:356)<br/>		at hudson.remoting.Channel.call(Channel.java:1001)<br/>		at hudson.FilePath.act(FilePath.java:1166)<br/>		at hudson.FilePath.act(FilePath.java:1155)<br/>		at hudson.tasks.junit.JUnitParser.parseResult(JUnitParser.java:107)<br/>		at hudson.tasks.junit.JUnitResultArchiver.parse(JUnitResultArchiver.java:153)<br/>		at hudson.tasks.junit.JUnitResultArchiver.parseAndSummarize(JUnitResultArchiver.java:247)<br/>		at hudson.tasks.junit.pipeline.JUnitResultsStepExecution.run(JUnitResultsStepExecution.java:63)<br/>		at hudson.tasks.junit.pipeline.JUnitResultsStepExecution.run(JUnitResultsStepExecution.java:29)<br/>		at org.jenkinsci.plugins.workflow.steps.SynchronousNonBlockingStepExecution.lambda$start$0(SynchronousNonBlockingStepExecution.java:47)<br/>		at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)<br/>hudson.AbortException: No test report files were found. Configuration error?<br/>	at hudson.tasks.junit.JUnitParser$ParseResultCallable.invoke(JUnitParser.java:154)<br/>	at hudson.FilePath$FileCallableWrapper.call(FilePath.java:3338)<br/>	at hudson.remoting.UserRequest.perform(UserRequest.java:211)<br/>	at hudson.remoting.UserRequest.perform(UserRequest.java:54)<br/>	at hudson.remoting.Request$2.run(Request.java:376)<br/>	at hudson.remoting.InterceptingExecutorService.lambda$wrap$0(InterceptingExecutorService.java:78)<br/>	at java.util.concurrent.FutureTask.run(FutureTask.java:266)<br/>	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)<br/>	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)<br/>	at java.lang.Thread.run(Thread.java:750)<br/><br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>[2022-08-11T09:11:33.380Z] + rm -rf console.log<br/>[Pipeline] sh<br/>
</details>

**Review by danielezonca — APPROVED** [review]

**Comment by danielezonca** [comment]
Jenkins kogito-examples failure is random
`[2022-08-11T09:11:32.840Z] [ERROR]  You have exceeded a secondary rate limit. Please wait a few minutes before you try again.`

Jenkins (re)run kogito-examples tests

**Review by evacchi — APPROVED** [review]
